docs/package.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "docs",
+    "version": "99.0.0",
```

Comment:
For the future: Workspaces need a version, docs however do not really have a version as they are ideally always at the head of the repository. This is therefore just a placeholder in lieu of a real version.

Reply:
@whoacowboy That sounds like a much more sane approach compared to this one. Thanks for reminding me to keep things simple.